wiki/Beatmaps/id.md

```diff
@@ -0,0 +1,77 @@
+Beatmap yang memenuhi syarat menggunakan ikon centang (![Check icon](/wiki/shared/status/qualified.png)) di halaman pemilihan lagu.
+
+Beatmap ini berada di tahap terakhir dari proses beatmapping dan modding. Sebelum beatmap mulai, dialog akan ditampilkan, memperingati status dari beatmap akan menyebabkan:
+
+- Tidak ada [pp](/wiki/pp) yang akan diberikan.
+- Semua skor akan dihapus saat status beatmap diganti.
+  - Jika status beatmap diganti ke ranked, approved, atau kembali ke pending; semua skor akan dihapus!
+  
+Ketika sebuah beatmap memenuhi syarat,telah lulus [kriteria peringkatan](/wiki/ranking_criteria) dan telah diperiksa oleh dua [Beatmap Nominators](/wiki/Beatmap_Nominators). Beatmap yang memenuhi syarat memasuki tahap terakhir pertahanan karena mereka harus berada dalam status ini selama satu minggu sebelum menjadi [ranked](#ranked) atau [approved](#approved) secara permanen.
+ 
+Namun, selama durasi satu minggu, [QATs](/wiki/QAT) diizinkan untuk mencari permasalahan di beatmap ini. Jika hal ini terjadi, status beatmapset akan kembali ke pending dan prosesnya akan diulangi lagi.
+  
+### Loved
+  
+Beatmap yang dicintai menggunakan ikon hati (![Heart icon](/wiki/shared/status/loved.png)) di halaman pemilihan lagu.
+  
+Beatmap ini mungkin tidak memenuhi [kriteria peringkatan](/wiki/ranking_criteria) tetapi dicintai oleh komunitas dan secara teratur ditambahkan berdasarkan suara. Sebelum beatmap mulai, dialog akan ditampilkan, memperingati status dari beatmap akan menyebabkan:
+
+- Tidak ada [pp](/wiki/pp) yang akan diberikan.
+- Semua skor akan dihapus saat pembuat memperbaharui beatmapnya.
+
+Status ini akan memberikan leaderboard pada beatmap
+
+Untuk menjadikan sebuah beatmap ke status dicintai, lihat kiriman forum: [Get your beatmap Loved!](https://osu.ppy.sh/forum/t/549835).
+
+### Pending
+
+*Lihat juga: [Beatmap Ranking Procedure](/wiki/Beatmap_Ranking_Procedure)*
+
+Beatmap yang pending menggunakan ikon tanda tanya (![Question mark icon](/wiki/shared/status/pending.png)) di halaman pemilihan lagu.
+
+Sebuah beatmap dengan status pending biasanya menunggu untuk di [modded](/wiki/modded). [Pembuat](/wiki/Creators) harus mencari modder dari komunitas dan setidaknya dua Beatmap Nominator untuk mengeluarkan beatmapset mereka dari status ini.
+
+Beatmap yang tidak akif yang telah ada di kategori ini selama 4 minggu akan dipindahkan ke status [graveyard](#graveyard) secara otomatis.
+
+Semua beatmap yang pending tidak mempengaruhi statistik pengguna.
+
+### Graveyard
+
+Beatmap Graveyard menggunakan ikon tanda tanya (![Question mark icon](/wiki/shared/status/graveyard.png)) di halaman pemilihan lagu.
+
+Beatmap tersebut sebenarnya dalam status [pending](#pending), tetapi dipindahkan ke status ini karena belum diperbarui 
+atau post nya tidak aktif selama 4 minggu. Beatmap ini tidak dapat diperbarui sampai [mapper](/wiki/mapper) menghidupkannya dengan mengklik
+tautan `Revive Beatmap` di profil mereka, di bawah kategori `Beatmaps`.
```

Comment:
Kategori `Beatmap`
We're using Bahasa Indonesia Website remember, as of Revive Beatmap i can't find it